conductor_api/src/conductor/base.rs

```diff
@@ -626,6 +626,59 @@ impl Conductor {
                     ))
                 })?;
 
+                // This is where we are checking the consistency between DNAs: for now we compare
+                // the hash provided in the TOML Conductor config file with the computed hash of
+                // the loaded dna.
+                {
+                    let dna_hash_from_conductor_config =
+                        HashString::from(dna_config.hash.clone()
+                                         // .unwrap_or("Missing DNA hash from Conductor config.".to_string()))
+                                         .unwrap_or("".to_string()))
+                                         .to_owned();
+                    let dna_hash_computed = &dna.address();
+
+                    // Let's warn the user that we encounter an empty DNA hash in the Conductor
+                    // config that is mmeant to be skipped by the consistency checking mechanisim
+                    context.log("warn/Conductor: Empty DNA hash encountered in the Conductor configuration.");
```

Comment:
This warning seems to get logged unconditionally, no?

Reply:
Oups, my bad! Good catch!